src/ctree-app/ctree-app.html

```diff
@@ -411,9 +432,19 @@
           if (!pageData.icon.startsWith(this.rootPath)) {
             pageData.icon = this.rootPath + pageData.icon;
           }
-          newNavList.push({name: name, data: pageData});
+
+          if(this.searchText){
+            if(pageData.label.toLowerCase().indexOf(this.searchText.toLowerCase()) !== -1){
```

Comment:
We may want to get the lower case search text before we start the for loop so we don't need to regenerate it for each tree.